### ISSUE TITLE
coco dataset pretraining

### ISSUE THREAD

**Issue by gaopeng-eugene** [DrSleep/tensorflow-deeplab-resnet#48]
Recently, I am trying to add coco dataset into this framework which is a common trick for many state-of-the-art semantic segmentation method. I will try to add the code as soon as possible.

**Comment by arslan-chaudhry** [comment]
If I am not mistaken *init.caffemodel* which is converted to *deeplab_resnet_init.ckpt* is already pre-trained on MS-COCO for semantic segmentation task. @DrSleep can you please confirm that. Also, do you have resenet init model trained for classification task converted to tf *.ckpt* format somewhere. If not, do you know if Deeplab folks have released any such init model? I'm trying to replicate the results for weakly supervised setting and I don't want pre-training on fully supervised coco dataset.

**Comment by DrSleep** [comment]
Yes, it is pre-trained on COCO for sem. segmenation.

I don't have the classification net in ckpt, unfortunately.
DeepLab folks have been using the files provided by K.He pre-trained on
ImageNet. I think you may use them as well with slight changes in prototxt
if necessary.

**Comment by bhack** [comment]
What about [sceneparsing] (http://sceneparsing.csail.mit.edu)?

**Comment by gaopeng-eugene** [comment]
I think you are right. Joint training COCO and PASCAL VOC give almost the same performance.

**Comment by arslan-chaudhry** [comment]
@gaopeng-eugene if you have the segmentation masks and class categories of coco dataset extracted somewhere from json files, could you please share those?
